app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "new-cards@flashcards.com"
+  layout 'mailer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.